tests/base.py

```diff
@@ -305,8 +312,13 @@ def compare(self):
                 'baseline image does not exist:\n{}'.format(os.path.relpath(expected))
             )
 
+        if os.name == "nt" and self.windows_tol is not None:
+            err_tol = self.windows_tol
+        else:
+            err_tol = self.tol
+
         # Perform the comparison
-        err = compare_images(expected, actual, self.tol, in_decorator=True)
+        err = compare_images(expected, actual, err_tol, in_decorator=True)
```

Comment:
If self.tol is updated in the init method, this change will not be needed.

Reply:
great idea!